Fabric.Terminology.API/Models/CompareValueSetsQuery.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Fabric.Terminology.API.Models
+{
+    using System;
+
+    using Newtonsoft.Json;
+
+    public class CompareValueSetsQuery : MultipleCodeSystemQuery
+    {
+        [JsonProperty("valueSetGuids")]
```

Comment:
Rather than using JsonProperty all over the place, can you just configure the serializer to use a CamelCase formatter?

Reply:
These are only there so the Swagger docs render the input model correctly.  There is a task to move the swagger UI to a different project - but I've not prioritized that as of yet.

Reply:
You're using Swashbuckle, right? From what I can tell, it should pick up the global JSON config settings, so if you're using a CamelCasePropertyNamesContractResolver it should render these as camel case automatically.